youtube_dl/extractor/yahoo.py

```diff
@@ -556,3 +560,285 @@ def _real_extract(self, url):
+            or self._html_search_regex('<title>([^<]+)</title>', webpage, 'title')
+        )
+        description = (
+            self._og_search_description(webpage, default=None)
+            or self._html_search_meta('description', webpage, default=None)
+            or self._html_search_meta('twitter:description', webpage, default=None)
+        )
+        thumbnail = self._og_search_thumbnail(
+            webpage, default=None
+        ) or self._html_search_meta('twitter:image', webpage, default=None)
+
+        if display_id == host:
+            # Headline page with playlist of videos
+            formats = []
+
+            # No single useful `contentid` on page
+            content_id = display_id
+
+            stream_urls_plists = re.findall(
+                r'YJNEWS\.LIVESTREAM\.SRC.+=(?:\s+)?["\'](?P<streamurl>[^"\']+plist=(?P<plist>\d+)[^"\']+)',
+                webpage,
+            )
+            if not stream_urls_plists:
+                # Manually build stream urls ('news.yahoo.co.jp', ...)
+                sbase_url = 'https://s.yimg.jp/images/media/video/player/html/embed_1.2.5.html?service=news&service_type=video&plist=%s&poster=%s&sp=%s&country=JP'
+
+                params = re.findall(
+                    r'["\']plist["\']:\s*["\'](?P<plist>[^"\']+)[^;]+?["\']poster["\']:\s*["\'](?P<poster>[^"\']+)[^;]+?["\']spaceId["\']:\s*["\'](?P<spaceid>[^"\']+)',
+                    webpage,
+                )
+                for p in params:
+                    stream_urls_plists.append((sbase_url % p, p[0]))
+
+            for url, plist_id in stream_urls_plists:
+                embed_page = self._download_webpage(
+                    'https://s.yimg.jp/images/media/video/player/js/embed_1.0.9.min.js',
+                    plist_id,
+                    headers={
+                        'Authority': 's.yimg.jp',
+                        'Path': '/images/media/video/player/js/embed_1.0.9.min.js',
+                        'Referer': url,
+                        'User-Agent': self._USER_AGENT,
+                    },
+                )
+                account = self._search_regex(
+                    r'\w+\.video_attr\[(["\'])data-account\1\]\s*=\s*\1(?P<accountid>[\d]+)',
+                    embed_page,
+                    'account id',
+                    group='accountid',
+                )
+                player = self._search_regex(
+                    r'\w+\.video_attr\[(["\'])data-player\1\]\s*=\s*\1(?P<playerid>[^"\']+)',
+                    embed_page,
+                    'player id',
+                    group='playerid',
+                )
+                embed = self._search_regex(
+                    r'(["\'])data-embed\1\s*:\s*\1(?P<embed>[^"\']+)',
+                    embed_page,
+                    'embed',
+                    group='embed',
+                )
+                indexjs_page = self._download_webpage(
+                    'https://players.brightcove.net/%s/%s_%s/index.min.js'
```

Comment:
This must be delegated to brightcove extractor.

Reply:
I tried to delegate to Brightcove extractor but it failed because this is a playlist of videos (not supported by extractor) on the headline pages, not a single video. Please advise.

Reply:
@dstftw I'm going to add support for playlists to BrightcoveNewIE in a separate PR. If that is accepted, then I can delegate this part to the BrightcoveNewIE extractor.